rusk/src/lib/lib.rs

```diff
@@ -8,6 +8,7 @@ use tracing::info;
 pub mod encoding;
 pub mod services;
 pub mod transaction;
+pub mod schedule_loader;
```

Comment:
I think we could just have a `schedule` module, and having the `Loader` exposed (basically `schedule::Loader`)